File: src/main/java/me/lulu/hypixelutilities/craft/gui/Range.java
package me.lulu.hypixelutilities.craft.gui;

public class Range {
    private int start;
    private int end;

    public Range(int start, int end) {
        this.start = start;
        this.end = end;
    }

    public boolean isInRange(int i) {
        return i >= start && i < end;
    }
}

File: src/main/java/me/lulu/hypixelutilities/craft/gui/Range.kt
package me.lulu.hypixelutilities.craft.gui

class Range(private val startX: Int, private val startY: Int, private val endX: Int, private val endY: Int) {
    fun isInRange(x: Int, y: Int): Boolean {
        return x in startX until endX && y in startY until endY
    }
}
